import.py

```diff
@@ -145,15 +158,14 @@ def fixup_build_command(
 
     return res, assembler
 
-
```

Comment:
fwiw this newline is due to `black` auto-formatting; it'll get reinserted the next time I run that. Don't worry about it though